memgpt/constants.py

```diff
@@ -1,6 +1,6 @@
 import os
 
-MEMGPT_DIR = os.path.join(os.path.expanduser("~"), ".memgpt")
+MEMGPT_DIR = os.getenv(os.path.join("MEMGPT_CONFIG_PATH",".memgpt"), default=os.path.join(os.path.expanduser("~"), ".memgpt"))

```

Comment:
I think `MEMGPT_CONFIG_PATH` is currently the actual path of the config? so `~/.memgpt/config`, rather than the directory. 

So this should maybe be: 

```
MEMGPT_DIR = os.getenv("MEMGPT_CONFIG_PATH", default=os.path.join(os.path.expanduser("~"), ".memgpt"))
```

Reply:
The MEMGPT_DIR was not supposed to be included only the one for .gitignore.